.github/workflows/close-stalled.yml

```diff
@@ -3,6 +3,11 @@ on:
   schedule:
     - cron: "0 0 * * *"
 
+env:
+  CLOSE_MESSAGE: >
+    Closing this because it has stalled. Feel free to reopen if this issue is still relevant,
```

Comment:
This uses "issue" also for PRs, is that wanted?

Reply:
I didn't see that it was different. Fixed, thank you!